src/components/YearDetailsPanel/year-details-panel-view.js

```diff
@@ -0,0 +1,77 @@
+import React from "react";
+import style from "../SideBar/details-panel.module.css";
+import { ReactComponent as Exit } from "../../assets/Exit.svg";
+import { getFieldColor } from "../../util/utility";
+
+const YearDetailsPanel = props => {
+  const color = props.year.forschungsbereich
+    ? getFieldColor(props.year.forschungsbereich)
+    : "#aaa";
+  return (
+    <div className={style.DetailsWrapper}>
+      <div
+        className={style.DetailsTitle}
+        style={{
+          borderBottomColor: color,
+          color: color
+        }}
+      >
+        <div className={style.DetailsExit} onClick={props.returnToFilterView}>
+          <Exit height={35} width={35} />
+        </div>
+        <span className={style.titleTopic}>Jahr {props.year.year}</span> <br />
+        <span className={style.titleText}>
+          {props.year.targetgroup
+            ? props.year.targetgroup
+            : props.year.forschungsbereich}
+        </span>
+      </div>
```

Comment:
looks very cool !

Reply:
however, we have a problem. 

You save the entire year object on selection. 
This doesn't play nice with url mapping (makes state string very long).
See if you can save a small reference (year plus number of field)  and then recreate the object from the reference where needed

Reply:
replaced with "year|fb/targetgroup"